examples/spark-sensors-proto/project/cloudflow-plugins.sbt

```diff
@@ -0,0 +1,5 @@
+// Resolver for the cloudflow-sbt plugin
+//
+resolvers += Resolver.url("cloudflow", url("https://lightbend.bintray.com/cloudflow"))(Resolver.ivyStylePatterns)
```

Comment:
This is not necessary, sbt-cloudflow is published to sonatype.

Reply:
Fixed